gadget/gadget-amd64.yaml

```diff
@@ -25,12 +25,22 @@ volumes:
         type: C12A7328-F81F-11D2-BA4B-00A0C93EC93B
         size: 1200M
         update:
-          edition: 2
+          edition: 3
         content:
+          - source: shim.efi.signed
+            target: EFI/BOOT/BOOTX64.efi
+          - source: fbx64.efi
+            target: EFI/BOOT/fbx64.efi
           - source: grubx64.efi
-            target: EFI/boot/grubx64.efi
+            target: EFI/BOOT/grubx64.efi
```

Comment:
Are we sure we need grub in `EFI/BOOT`? Afaiu reading shim code, shim would never run it if it finds `EFI/BOOT/fbx64.efi`

Reply:
I think you are right.